examples/RDF_to_LPG_Example.ipynb

```diff
@@ -0,0 +1,266 @@
+   "outputs": [],
+   "source": [
+    "b = \"\"\n",
+    "pma.populate_predicates(b)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "predicates_to_collapse = [_.uri for _ in pma.multiselect.selected_things_list]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 2. Now we can call the `ipyradiant` fuction `collapse_predicates` that will collapse down the data in the graph based on the specified `predicates_to_collapse`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# for the `collapse_predicates` function, we will use the same namespaces as we did with the `PredicateMultiselectApp`\n",
+    "netx_graph = collapse_predicates(\n",
+    "    graph, predicates_to_collapse, namespaces=pma.namespaces\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evidence of Correct Operation\n",
+    "\n",
+    "Here we can see what the first node in the graph looks like now. We can see that\n",
+    "`rdfs:comment`, a previous predicate, has been collapsed onto the node and the\n",
+    "corresponding object, \"A object of the relationship is a copy of the Source resource's\n",
+    "representation, appropriate for the Annotation.\" is attached to it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "netx_graph.nodes[rdflib.term.URIRef(\"http://www.w3.org/ns/oa#cachedSource\")]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With this collapsing algorithm, if an object is a `URIRef` as well as a subject of a\n",
+    "different object in the graph, it will not be collapsed. However, the data will still be\n",
+    "moved onto the subject node if the user wishes to collapse said predicate. If the object\n",
+    "is a literal, then that node will disappear from the graph and the predicate will be\n",
+    "collapsed onto the subject, with all of the data being moved onto the node and erased\n",
+    "from the graph. Examples of these behaviors are seen below:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will focus on two triples.. The first is\n",
+    "`(rdflib.term.URIRef('http://www.w3.org/ns/oa#Motivation'), rdflib.term.URIRef('http://www.w3.org/2000/01/rdf-schema#subClassOf'), rdflib.term.URIRef('http://www.w3.org/2004/02/skos/core#Concept'))`.\n",
+    "Since this object is not a subject, when we collapse this predicate we can expect the\n",
+    "node `rdflib.term.URIRef('http://www.w3.org/2000/01/rdf-schema#subClassOf'` to\n",
+    "disappear."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "example_1 = collapse_predicates(\n",
```

Comment:
This example needs a rework. The node that is checked should be the object not the predicate.